tests/test_integration.py

```diff
@@ -454,37 +488,61 @@ def get_existing():
             ).order_by(
                 models.ScheduleABySize.committee_id,
             ).first()
+        EXISTING_RECEIPT_AMOUNT = 504
+        filing = self.NmlSchedAFactory(
+            rpt_yr=2015,
+            cmte_id='X1234',
+            contb_receipt_amt=EXISTING_RECEIPT_AMOUNT,
+            contb_receipt_dt=datetime.datetime(2015, 1, 1),
+            receipt_tp='15J',
+        )
+        self.FItemReceiptOrExp(
+            sub_id=filing.sub_id,
+            rpt_yr=2015,
+        )
+        db.session.commit()
+        manage.update_aggregates()
+        db.session.execute('refresh materialized view ofec_sched_a_aggregate_size_merged_mv')
+        self._clear_sched_a_queues()
         existing = get_existing()
         total = existing.total
         count = existing.count
-        self.SchedAFactory(
+        NEW_RECEIPT_AMOUNT = 538
```

Comment:
Out of curiosity, why pull this and `EXISTING_RECEIPT_AMOUNT` out as separate variables?  Ease of change and maintenance later on?  Totally fine, I just wanted to make sure I understand!